### ISSUE TITLE
Limit the download speed of certain users

### ISSUE THREAD

**Issue by nintiendo1** [LANCommander/LANCommander#84]
Because I have a speed limit on my server of 1 gbps, I don't want to have a download speed limit for download games, but I don't want everyone to be able to download games at maximum speed because then they would take up all the bandwidth on the server.
Thanks.

**Comment by pathartl** [comment]
Good idea. Will take some time and planning to implement. Users should also be able to define a limit in the launcher, so a server and launcher implementation will be required.

For my own notes:
https://github.com/dotnet/aspnetcore/issues/45298#issuecomment-1761536734

**Comment by pathartl** [comment]
🤖 **Board Manager — Preliminary Triage**

**Component classification:** `Server`

This issue has been added to the [Development Tracker](https://github.com/orgs/LANCommander/projects/1).

*This comment was added automatically. Status will be updated as commits land.*
